ee/docker-ee/ubuntu.md

```diff
@@ -23,73 +23,58 @@ Docker Engine - Community users should go to
 access to Docker Engine - Enterprise for all Linux distributions. To get this
 information, do the following:
 
-- Go to [https://hub.docker.com/my-content](https://hub.docker.com/my-content).
-- Each subscription or trial you have access to is listed. Click the **Setup**
+1. Go to [https://hub.docker.com/my-content](https://hub.docker.com/my-content).
+2. Each subscription or trial you have access to is listed. Click the **Setup**
   button for **Docker Enterprise Edition for Ubuntu**.
-- Copy the URL from the field labeled
+3. Copy the URL from the field labeled
   **Copy and paste this URL to download your Edition**.
 
 Use this URL when you see the placeholder text `<DOCKER-EE-URL>`.
 
-To learn more about Docker Enterprise, see
+To learn more about Docker Engine - Enterprise, see
 [Docker Enterprise Edition](https://www.docker.com/enterprise-edition/){: target="_blank" class="_" }.
 
 ### OS requirements
 
 To learn more about software requirements and supported storage drivers,
 check the [compatibility matrix](https://success.docker.com/article/compatibility-matrix).
 
-> **Note:**
-> IBM Z (`s390x`) is supported for Docker Engine - Enterprise 17.06.xx only.
+> **Note**
+> 
+> IBM Z (`s390x`) is supported for Docker Engine - Enterprise 17.04.xx only.
 
 ### Uninstall old versions
 
-Older versions of Docker were called `docker` or `docker-engine`. In addition,
-if you are upgrading from Docker Engine - Community to Docker Engine -
-Enterprise, remove the Docker Engine - Community package.
+Use the `apt-get remove` command to uninstall older versions of Docker Engine - Enterprise (called `docker` or `docker-engine`). In addition,
+if you are upgrading from Docker Engine - Community to Docker Engine - Enterprise, remove the Docker Engine - Community package.
 
 ```bash
 $ sudo apt-get remove docker docker-engine docker-ce docker-ce-cli docker.io
 ```
 
-It's OK if `apt-get` reports that none of these packages are installed.
+The `apt-get` command may report that none of the packages are installed.
 
-The contents of `/var/lib/docker/`, including images, containers, volumes, and
-networks, are preserved. The Docker Engine - Enterprise package is now called
-`docker-ee`.
+> **Note**
+>
+> The contents of `/var/lib/docker/`, including images, containers, volumes, and networks, are preserved. The Docker Engine - Enterprise package is now called `docker-ee`.
 
 #### Extra steps for aufs
 
 For Ubuntu 16.04 and higher, the Linux kernel includes support for overlay2,
-and Docker Engine - Enterprise uses it as the default storage driver. If you
-need to use `aufs` instead, you need to configure it manually.
-See [aufs](/engine/userguide/storagedriver/aufs-driver.md).
+and Docker Engine - Enterprise uses it as the default storage driver. If you need to use aufs instead, be aware that it must be manually configured (refer to [aufs](/engine/userguide/storagedriver/aufs-driver.md)).
 
```

Comment:
Instead of "See aufs." at the end, can we just make the reference to it in the sentence a link?